astropy/units/tests/test_units.py

```diff
@@ -348,10 +348,9 @@ def test_compose_roundtrip(unit):
 @pytest.mark.parametrize('unit', sorted(COMPOSE_CGS_TO_SI, key=_unit_as_str),
                          ids=_unit_as_str)
 def test_compose_cgs_to_si(unit):
-    for iter in range(10):
-        si = unit.to_system(u.si)
-        assert [x.is_equivalent(unit) for x in si]
-        assert si[0] == unit.si
+    si = unit.to_system(u.si)
+    assert [x.is_equivalent(unit) for x in si]
+    assert si[0] == unit.si
```

Comment:
Wait what, why was this in a loop??

Reply:
I second this confusion!

Reply:
this was added as part of #4849, which was to speed up the unit tests! Must have been some oversight.